x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/10_semantic_text_field_mapping.yml

```diff
@@ -105,6 +159,55 @@ setup:
   - match: { "test-index.mappings.properties.dense_field.model_settings.task_type": text_embedding }
   - length: { "test-index.mappings.properties.dense_field": 3 }
 
+---
+"Add field_caps support for semantic_text field types with text embedding":
+
+  - do:
+      field_caps:
+        include_empty_fields: false
```

Comment:
Please add a test with include_empty_fields set to true to ensure that the field is returned when it is not yet populated.